api/project/models/project.js

```diff
@@ -5,4 +5,30 @@
  * to customize this model
  */
 
-module.exports = {};
+module.exports = {
+  /**
+  * Triggered before user creation.
+  */
+  lifecycles: {
+    async afterCreate(result, data) {
+
+      const { title, description, team } = data;
+
+      const group = await strapi.services['google-manager'].createGroup(description, title);
+
+      await strapi.services.project.giveTeamGroup(team, group);
+
+      //Lets strapi-svc@devlaunchers.com be owner of the google group to fix google meets auto admit problem
+      await strapi.services['google-manager'].joinGroup(group.id, process.env.DEVLAUNCHERS_GOOGLE_DIRECTORY_JWT_SUBJECT, 'OWNER');
+
+
+      const calendar = await strapi.services['google-manager'].createCalendar(title);
+
+      result.calendarId = calendar.id;
```

Comment:
result is a parameter of the function. Will the caller of `afterCreate` see the added field?